src/gsi-ncdiag/gsi_ncdiag.py

```diff
@@ -187,6 +189,7 @@
 gsi_add_vars = {
     'ObsBias': 'GsiObsBias',
     'Observation_Type': 'ObsType',
+    'Observation_Subtype': 'ObsSubType',
```

Comment:
Just a question, does this follow the convention that we will change to in 2 weeks time?

Reply:
Not sure. I have no knowledge about that.  We need those subtypes for our configuration.

Reply:
This is a variable name I placed into the `ObsSpace.yaml` file that we will be using soon to validate that incoming data has all the variable names to match the JEDI Data Conventions.  Please use this: `observationSubTypeNum`